.vib/consul/goss/goss.yaml

```diff
@@ -0,0 +1,44 @@
+port:
+  tcp6:8400:
```

Comment:
Instead of a fixed `8400` value for the port, you may use the specific env. var. to adapt to changes